ros2run/ros2run/command/run.py

```diff
@@ -29,8 +29,8 @@ class RunCommand(CommandExtension):
     def add_arguments(self, parser, cli_name):
         arg = parser.add_argument(
             '--prefix',
-            help="Prefix command, which should go before the executable. "
-                 "Command must be wrapped in quotes if it contains spaces "
+            help='Prefix command, which should go before the executable. '
+                 'Command must be wrapped in quotes if it contains spaces '
```

Comment:
looks a bit weird to have part of the multiline string in simple quotes and parts in double quotes. Is there an alternative allowing us to stay consistent regarding quoting within the same string?

Reply:
> Is there an alternative allowing us to stay consistent regarding quoting within the same string?

No, each string is checked on its own.

Reply:
too bad, lgtm then